common-lib/common/panels/stat/base.libsonnet

```diff
@@ -0,0 +1,10 @@
+local g = import '../../g.libsonnet';
+local stat = g.panel.stat;
+local base = import '../all/base.libsonnet';
+
+stat + base {
+  new(title, targets, description=''):
+    stat.new(title)
+    + super.new(title, targets, description),
```

Comment:
`title` doesn't seem to ever be used in the base class, since the panel type is not yet known.

Probably makes sense to drop title from the all panels base.

Reply:
Dropped!